src/Model/TransportModel/tsp1apt1.f90

```diff
@@ -2381,27 +2273,27 @@ subroutine apt_fill_budobj(this, x)
         call this%apt_accumulate_ccterm(n1, q, ccratin, ccratout)
       end do
     end if
-
+    !
     ! -- FROM MOVER
     if (this%idxbudfmvr /= 0) then
       idx = idx + 1
       nlist = this%ncv
       call this%budobj%budterm(idx)%reset(nlist)
-      do n1 = 1, nlist
-        q = this%qmfrommvr(n1)
+      do j = 1, nlist
+        call this%apt_fmvr_term(j, n1, n2, q) ! kluge note: don't really need to do this in apt_fmvr_term now, since no override by uze
```

Comment:
Are these changes ready for develop?

Reply:
After discussing this over with @aprovost-usgs, we concluded that it is ready for the develop branch. In essence, several lines were packaged into this subroutine (`apt_fmvr_term`). The "kluge note" has been removed.